ddp-workspace/projects/ddp-osteo/src/styles.scss

```diff
@@ -58,6 +58,10 @@ ul {
     margin: 0;
 }
 
+.maincolor {
```

Comment:
Please write more descriptive class name, E.g: .emailColor;

Reply:
it's fixed